index.js

```diff
@@ -199,7 +199,11 @@ function hash(src, filePath) {
 }
 
 function symlinkOrCopyFromCache(entry, dest, relativePath) {
-  mkdirp.sync(path.dirname(entry.outputFile));
-
-  symlinkOrCopySync(entry.cacheFile, dest + '/' + relativePath);
+  try {
```

Comment:
@hjdivad i think the next step (After this PR) is to use FSTreeDiff stuff to sort out all the instability issues. I believe this is intended as a  quickfix.